src/Helpers/GetDefinedFilters.ts

```diff
@@ -0,0 +1,14 @@
+// tslint:disable-next-line
+import '../Extensions/Array'
+
+interface Query {
```

Comment:
`Query` says more about _how_ we're using this interface than _what_ it is. Maybe simply calling it `WorksFilters`?